v1/lib/core/Site.js

```diff
@@ -107,6 +107,21 @@ class Site extends React.Component {
               }}
             />
           )}
+          {this.props.config.algolia && (
+            <script
+              dangerouslySetInnerHTML={{
+                __html: `
+                document.addEventListener('keyup', function(e) {
+                  // keyCode for '/' (slash)
```

Comment:
Add 

```
if (e.target !== document.body) { 
  return; 
}
```
here so that we only focus on the search bar when no other items on the page are in focus.

Try your current script vs the script with the `e.target` checking on https://reasonml.github.io/en/try and you'll see why it's important to add that line.

Reply:
Perfect - I thought that might be an issue. Thanks!